grouper/model_soup.py

```diff
@@ -66,7 +66,7 @@
 )
 OWNER_ROLE_INDICES = set([GROUP_EDGE_ROLES.index("owner"), GROUP_EDGE_ROLES.index("np-owner")])
 APPROVER_ROLE_INDICIES = set([GROUP_EDGE_ROLES.index("owner"), GROUP_EDGE_ROLES.index("np-owner"),
-        GROUP_EDGE_ROLES.index("manager")])
+        GROUP_EDGE_ROLES.index("manager")])  # TODO: Fix spelling
```

Comment:
The comment is supposed to be for the line above, APPROVER_ROLE_INDICIES has indices misspelled.